quicklogic/qlf_k4n8/utils/pinmap_parse.py

```diff
@@ -0,0 +1,348 @@
+#!/usr/bin/env python3
+import argparse
```

Comment:
Please add root level docstring explaining what is in here.

Reply:
Comment added.